static/script/devices/mediaplayer/samsung_streaming.js

```diff
@@ -95,14 +103,16 @@ require.def(
                     this._registerEventHandlers();
```

Comment:
That shouldn't happen, did you try and declare the variable outside of the init and then put the command to acquire the logger in init?